### PR TITLE
[ICDS] Fixed problem with broken zip file

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz @Rohit25negi 

I fixed a problem with the broken zip file and also I added a message if the file doesn't exist to don't throw a 500 error.
TICKET: https://dimagi-dev.atlassian.net/browse/QA-138

I think this is ready to merge to unblock QA process.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

**Comment by lwyszomi** [comment]
cas_data_export

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -1729,17 +1731,50 @@ def valid_state_names(self):
...
+        year = int(request.GET.get('year', None))
+        selected_date = date(year, month, 1).strftime('%Y-%m-%d')
+
+        sync, blob_id = get_cas_data_blob_file(data_type, state_id, selected_date)
 
         zip_name = 'cas_data_%s' % sync.file_added.strftime('%Y-%m-%d')
         try:
-            return export_response(sync.get_file_from_blobdb(), 'csv', zip_name)
+            output = BytesIO()
+            f = zipfile.ZipFile(output, 'w', zipfile.ZIP_DEFLATED)
```
Comment (calellowitz) [inline custom/icds_reports/views.py]: What does this change by switching from `export_response`?  Also, why is the zipfile necessary? It is already a single file?

**Review by calellowitz — CHANGES_REQUESTED** [review]
couple small questions, but mostly looks fine.

**Comment by lwyszomi** [comment]
@calellowitz I updated the PR

**Comment by lwyszomi** [comment]
@calellowitz bump

**Review by calellowitz — APPROVED** [review]

**Pull request merged**